code/modules/clothing/modular_armor/attachments/cape.dm

```diff
@@ -0,0 +1,90 @@
+/obj/item/armor_module/armor/cape
+	name = "6E Chameleon Cape (Long)"
+	desc = "To improve on the design of the 7E badge, this cape is capable of two colors, for all your fashion needs. It also is equiped with thermal insulators so it will double as a blanket. \n Attack with green facepaint to color. Attaches onto a uniform. Activate it to toggle the hood."
```

Comment:
```suggestion
	desc = "A chromatic cape to improve on the design of the 7E badge, this cape is capable of two colors, for all your fashion needs. It also is equipped with thermal insulators so it will double as a blanket. \n Interact with facepaint to color. Attaches onto a uniform. Activate it to toggle the hood."
```